librz/cons/histogram.c

```diff
@@ -207,3 +210,133 @@ RZ_API RZ_OWN RzStrBuf *rz_histogram_vertical(RZ_NONNULL RzHistogramOptions *opt
 	}
 	return buf;
 }
+
+RZ_API RzHistogramOptions *rz_cons_histogram_options_new() {
+	RzHistogramOptions *histops = RZ_NEW0(RzHistogramOptions);
+	if (!histops) {
+		return NULL;
+	}
+	return histops;
+}
+RZ_API void rz_cons_histogram_options_free(RzHistogramOptions *histops) {
```

Comment:
please add a new line between functions and add to all the new `RZ_API` methods a doxygen description.
See line `10` for an example.